packages/types/src/signInFuture.ts

```diff
@@ -35,6 +36,11 @@ export interface SignInFutureEmailCodeSendParams {
   email: string;
 }
 
+export interface SignInFutureEmailLinkSendParams {
+  email: string;
+  redirectUrl: string;
```

Comment:
I think we should update this to `verifyUrl`/`verificationUrl` instead - `redirectUrl` is overloaded and usually indicates the URL to redirect to after a complete si/su attempt